core/domain/config_services_test.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Testing Services for configuration properties."""
+
+import collections
+
+from core.domain import config_domain as cd
```

Comment:
Avoid using `as` since they distort readability.

Reply:
having hard time with line 62 - it is always greater than 80 characters.
I am trying to compare values before the change to reverted, if it did revert successfully.